letsencrypt/cli.py

```diff
@@ -537,6 +537,20 @@ def diagnose_configurator_problem(cfg_type, requested, plugins):
     raise errors.PluginSelectionError(msg)
 
 
+def _relevant(option):
```

Comment:
Since it's being called from other modules, this function should not be private (ie not start with underscore).